### PR TITLE
Prevent commented methods from being nominated

### LINKED ISSUE #25: Commented functions are being nominated
Functions that have comments, they are being caught by our engine to get documented

Try

```python
# def hello(x=1, y=2, z=3):
    # return 'Hello'
```

Output

Screen Shot 2019-05-09 at 5 02 03 PM

### PR DESCRIPTION
**Pull request by shanthanu9**
I think this fixes the bug. Do I have to write any tests for this?

Fixes #25

### CODE REVIEW

File: dyc/base.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def initialize(self, change=None):
                         if word.lstrip() in keywords
                     ]
                 )
-                > 0
+                > 0 and line.lstrip(' ')[0] != '#' # line is not a comment line
```
Comment (ronaldokun) [inline dyc/base.py]: > 0 and line.lstrip(' ').startswith("#")

Comment (Zarad1993) [reply dyc/base.py]: Hey guys, I think this attempt prevents the ability of identifying comments by other languages.

Preferable to have

in `config/defaults.yaml`
Add a `comments` list that takes as a start `['"""', '#', "'''"]`
Then update that here

Comment (ronaldokun) [reply dyc/base.py]: Yes. That would be a general solution.

**Comment by Zarad1993** [comment]
Solved in [43df5c36(https://github.com/Zarad1993/dyc/commit/43df5c3634e750cc4edf6e8735f31ac76156af21)

**Pull request closed without merging**